Dockerfile

```diff
@@ -20,7 +16,10 @@ RUN apt-get update -y && \
                        ruby2.5 ruby2.5-dev \
                        tzdata \
                        unattended-upgrades \
-                       update-notifier-common
+                       update-notifier-common \
+                       # needed to build some gem native extensions:
+                       libz-dev \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Just out of curiosity, what are the implications (e.g. on future uses of `apt`) of removing the files in this directory? Is this something we should have been doing all along, or is it a new requirement with 18.04?

Reply:
No, something we should have been doing all along. The implications are that `apt-get install` will just fail outright telling you to `apt-get update` instead of mysteriously trying and failing to download packages that no longer exist.